master/pkg/tasks/task.go

```diff
@@ -115,6 +115,9 @@ type TaskSpec struct {
 	Labels    []string
 	// Ports required by trial or commands and their respective base port values.
 	UniqueExposedPortRequests map[string]int
+
+	// For testing only.
```

Comment:
ditto here -- a change that's not related to k8s jobs should be called out somewhere or in its own PR

Reply:
need to make integration tests for jobs_test.go work much more simply (and faster)